src/licensedcode/data/rules/gpl-3.0-plus_22.yml

```diff
@@ -0,0 +1,2 @@
+licenses:
+    - gpl-3.0-plus

```

Comment:
@pombredanne  I felt the same. The second and third paragraph seems to be unique and it is mentioned in the `COPYING` file that the license is `GPL 3.0 or later`.